### PR TITLE
Add tests for decimal representation of very big and very small numbers

### PR DESCRIPTION
**Pull request by mr-c**
Closes: #240

### CODE REVIEW

**Comment by kinow** [comment]
Sorry was on mobile when leaving those comments and forgot to say that the tests for decimal look good to me. If the other changes are due to rebase/merge with other branches, we can merge and fix other things later :+1: Thanks!!

**Comment by mr-c** [comment]
> Sorry was on mobile when leaving those comments and forgot to say that the tests for decimal look good to me. If the other changes are due to rebase/merge with other branches, we can merge and fix other things later +1 Thanks!!

Which comments .. ?

**Comment by kinow** [comment]
Aaaahh, I never submitted the review :rofl:

File: salad/schema_salad/metaschema/salad.md [hunk 1]
```diff
@@ -72,7 +73,7 @@ documentation.
 
 ## Introduction to v1.1
 
-This is the third version of the Schema Salad specification.  It is
+This is the third version of of the Schema Salad specification.  It is
```
[the remark this answers is not in the document]
Comment (mr-c) [reply salad/schema_salad/metaschema/salad.md:76]: This was copied from https://github.com/common-workflow-language/schema_salad/tree/main/schema_salad/metaschema 🤠

**Pull request merged**